README.md

```diff
@@ -8,13 +8,13 @@ This is an API that [creates AWS (sub)accounts in an Organization](https://docs.
 
 1. Install system dependencies.
     1. [Go](https://golang.org/)
-    1. [Dep](https://golang.github.io/dep/docs/installation.html)
+    1. [Dep](https://golang.github.io/dep/docs/installation.html) (`go get -u github.com/golang/dep/cmd/dep`)
```

Comment:
Nitpick: I'd rather link to their install instructions than include something here that might get out of date.

Reply:
Generally, I agree.   There are multiple options to install, and the one I chose (download the binary package) didn't work.  I didn't try the `brew` install option, because I didn't remember if I had brew installed (I did, but it was broken), so the option that worked for me was the `go get` option.  The `go get` option has the advantage of it should work the same regardless of the platform you are on...the downside is the URL might change in the future and you aren't getting a released version.